universe/Tech.cpp

```diff
@@ -231,10 +231,21 @@ std::string Tech::Dump(unsigned short ntabs) const {
     std::string retval = DumpIndent(ntabs) + "Tech\n";
     retval += DumpIndent(ntabs+1) + "name = \"" + m_name + "\"\n";
     retval += DumpIndent(ntabs+1) + "description = \"" + m_description + "\"\n";
-    retval += DumpIndent(ntabs+1) + "shortdescription = \"" + m_short_description + "\"\n";
+    retval += DumpIndent(ntabs+1) + "short_description = \"" + m_short_description + "\"\n";
     retval += DumpIndent(ntabs+1) + "category = \"" + m_category + "\"\n";
     retval += DumpIndent(ntabs+1) + "researchcost = " + m_research_cost->Dump(ntabs+1) + "\n";
     retval += DumpIndent(ntabs+1) + "researchturns = " + m_research_turns->Dump(ntabs+1) + "\n";
+    retval += DumpIndent(ntabs+1) + "tags = ";
+    if (m_tags.empty()) {
```

Comment:
I think `tags = []` might not work to be parsed... can just leave out the tags line if there are none.

Reply:
Fixed.